### ISSUE TITLE
Toronto Songwriter / Performer Use Case: DID Method Candidates

### ISSUE THREAD

**Issue by mwherman2000** [decentralized-identity/did-methods#22]
Click here: [Toronto Songwriter / Performer Use Case: DID Method Candidates](https://www.youtube.com/watch?v=q-3GELg_WHE)

**Comment by stevenvegt** [comment]
I think you might confuse the entity with its identity. A DID method describes the technical method of (mostly) resolving the public keys on layer 1 of the trustOverIP model. What you seem to describe here are claims about the entities role in a system, in other wordt the identity of the entity. This concept lives on the third layer in the ToIP mode. An Identity claim can be issued by an authority by using Verifiable Credentials, e.g. create a MusicIndustryCredential with the claim "role".

So the DID method should only describe the mechanism for the Create, Resolve, Update and Deactivate operations and nothing about the actual identity of the entity.

**Comment by mwherman2000** [comment]
@stevenvegt I very much appreciate your analysis. This is where many people get confused about the nature of a DID (i.e. a character-string decentralized identifier).  

A DID is simply (a sometimes verifiable) name (identifier) for something (aka subject). A DID can name (reference, identify) Every Little Thing (#ELT) on the planet, every piece of knowledge, every bit of understanding, every ounce of wisdom (any subject). I made sure this was so when I contributed to the original [DID-CORE] specification - along with Drummond and others.

This diagram (with my highlighting is taken directly from [DID-CORE]. Aside from what's inside the green box, everything else is optional.

![image](https://github.com/user-attachments/assets/9248a3c4-5b4c-494d-a2e2-d0e55ab02c3f)

The #ToIP model has no bearing on [DID-CORE]. (I was part of that effort too.)

**Comment by peacekeeper** [comment]
> A DID is simply (a sometimes verifiable) name (identifier) for something (aka a subject). DIDs can name (reference, identify) Every Little Thing (#ELT) on the planet, every piece of knowledge, every bit of understanding, every ounce of wisdom (every possible subject)

I agree with this, but I also agree with @stevenvegt that "the DID method should only describe the mechanism for the Create, Resolve, Update and Deactivate operations and nothing about the actual identity of the entity."

DID methods were introduced to distinguish between different techical infrastructure that can be used for implementing DIDs, but this has nothing to do with what DIDs identify.

**Comment by mwherman2000** [comment]
> DID methods were introduced to distinguish between different techical infrastructure that can be used for implementing DIDs, but this has nothing to do with what DIDs identify.

@peacekeeper Markus can you quote a reference from [DID-CORE]?

...or is this folklore? We need to stay with the facts (as written in [DID-CORE]) - that's what specifications are for.

I was deeply involved in the [DID-CORE] and always viewed DID Methods as (abstract) classes of subjects/data types. There's a statement in the spec that says a DID Method isn't expected to be limited to a single underlying implementation (which is just the opposite).

**Comment by peacekeeper** [comment]
> @peacekeeper Markus can you quote a reference from [DID-CORE]?

I have provided references several times before, so please just see those other threads, or read the DID specification.

> I was deeply involved in the [DID-CORE] and always viewed DID Methods as (abstract) classes of subjects/data types.

I was much more involved in DID Core than you, and this interpretation has never even occurred to me. Please accept that this is NOT what DID methods are for.

**Comment by mwherman2000** [comment]
This is a professional forum and we shouldn't rely on individual options that are stated with no backup.

Your reply is an example of https://hyperonomy.com/2019/04/09/clique-speak/

**Comment by mwherman2000** [comment]
Re: This has never occurred to me before 

I have written extensively on this topic for many years (publicizing most if not all of this articles on the CCG making list:

- 2021 https://hyperonomy.com/2021/04/26/the-verifiable-economy-architecture-reference-model-ve-arm-fdo/
- 2021 https://hyperonomy.com/2021/07/26/bootstrapping-a-vdr-based-decentralized-credential-object-platform-von-example/
- 2021 https://hyperonomy.com/2021/06/15/the-verifiable-economy-fully-decentralized-object-fdo-example-bobs-udid-document/
- 2021 https://hyperonomy.com/2021/03/18/fully-decentralized-twitter-dewitter-app-scenario-platform-requirements-presentation/
- 2021 https://hyperonomy.com/2021/03/15/tdw-glossary-management-and-collaboration-platform-tdw-gmcp-digital-identity-neighborhood/
- 2021 https://hyperonomy.com/2018/12/21/decentralized-identifiers-dids-architecture-reference-model-arm/
- 2022 https://hyperonomy.com/2022/12/18/web-7-0-didcomm-agent-architecture-reference-model-didcomm-arm-0-40-december-18-2022/
- 2021 https://hyperonomy.com/2021/04/26/the-verifiable-economy-architecture-reference-model-ve-arm-fdo/

**Comment by mwherman2000** [comment]
Re: This has never occurred to me before 

I have written extensively on this topic for many years (publicizing most if not all of this articles on the CCG making list:

- 2021 https://hyperonomy.com/2021/04/26/the-verifiable-economy-architecture-reference-model-ve-arm-fdo/
- 2021 https://hyperonomy.com/2021/07/26/bootstrapping-a-vdr-based-decentralized-credential-object-platform-von-example/
- 2021 https://hyperonomy.com/2021/06/15/the-verifiable-economy-fully-decentralized-object-fdo-example-bobs-udid-document/
- 2021 https://hyperonomy.com/2021/03/18/fully-decentralized-twitter-dewitter-app-scenario-platform-requirements-presentation/
- 2021 https://hyperonomy.com/2021/03/15/tdw-glossary-management-and-collaboration-platform-tdw-gmcp-digital-identity-neighborhood/
- 2021 https://hyperonomy.com/2018/12/21/decentralized-identifiers-dids-architecture-reference-model-arm/
- 2022 https://hyperonomy.com/2022/12/18/web-7-0-didcomm-agent-architecture-reference-model-didcomm-arm-0-40-december-18-2022/
- 2021 https://hyperonomy.com/2021/04/26/the-verifiable-economy-architecture-reference-model-ve-arm-fdo/

**Comment by mwherman2000** [comment]
> So the DID method should only describe the mechanism for the Create, Resolve, Update and Deactivate operations and nothing about the actual identity of the entity.

This statement is not completely true.

The entire purpose of a subject's service endpoints are to enable interaction with the underlying entity (addressable by its DID). In Web 7.0, DIDScript(tm) exploits this concept:
- https://youtu.be/mf0aKLvJoCw?si=gCKSd8eBMujCgs0M
